src/components/MentorListDetails/MentorListDetails.jsx

```diff
@@ -1,68 +1,139 @@
 import { useState, setState, useEffect } from 'react'
 import useMentors from '../../hooks/use-mentors'
+import useSkills from "../../hooks/use-skills";
 import './MentorListDetails.css'
 
-function MentorListDetails({onEditMentorClick, editMentorOpen,  onChangeActiveMentor, activeMentor}) {
+// // --- Needs authentication handling to check token belongs to staff ---///
+
+function MentorListDetails({ onEditMentorClick, editMentorOpen, onChangeActiveMentor, activeMentor }) {
 
     const { mentors, isMentorsLoading, isMentorsError } = useMentors()
-    
-    // const [searchTermSkill, setSearchTermSkill] = useState("")
-    const [searchTermLocation, setSearchTermLocation] = useState("")
-    // const [searchTermRank, setSearchTermRank] = useState("")
+    const { skills, skillsLoading, skillsError } = useSkills([]);
+
+    // TODO Some of these should be grabbed from API (skills)
+    const status = ["Applied", "Validated", "Interviewed", "Ranked", "Accepted", "Onboarded", "Ready"]
+    const ranks = ["Junior", "Mid-level", "Lead"]
+    const locations = ["Brisbane", "Sydney", "Melbourne", "Adelaide", "Perth", "Canberra", "Darwin"]
+
+    const [searchTermStatus, setSearchTermStatus] = useState("All")
+    const [searchTermSkill, setSearchTermSkill] = useState("All")
+    const [searchTermLocation, setSearchTermLocation] = useState("All")
+    const [searchTermRank, setSearchTermRank] = useState("All")
 
-    // const handleChangeSkill = e => setSearchTermSkill(e.target.value)
-    const handleChangeLocation = e => setSearchTermLocation(e.target.value)
-    // const handleChangeRank = e => setSearchTermRank(e.target.value)
+    const [filteredMentors, updatFilteredMentors] = useState()
 
-    const handleOpenMentor = (event) =>{
+    const handleOpenMentor = (event) => {
         onChangeActiveMentor(event.target.value)
         onEditMentorClick("true")
-
     }
 
+
+    useEffect(() => {
+        updatFilteredMentors(mentors.filter(
+            mentor => (!mentor.is_superuser)
+        ))
+    }, [mentors])
+
+
+    useEffect(() => {
+
+        // Filtering the mentor list based on chosen selection in any of three filters - location/rank/skill
```

Comment:
Are we doing this in filter in multiple places? Should it be pulled out as a utility?

Reply:
Eventually I reckon yes - but I've put that in the post-MVP bucket of things to do 😆